docs/download/78.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: ICU 78
+nav_order: 898
+has_children: false
+parent: Downloading ICU
+---
+
+<!--
+© 2025 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU 78
+
+ICU is the [premier library for software internationalization](https://icu.unicode.org/#h.i33fakvpjb7o),
+used by a [wide array of companies and organizations](https://icu.unicode.org/#h.f9qwubthqabj).
+
+## Draft page
+
+**This is an early draft of the ICU 78 download page. Nothing is published yet.**
+
+## Release Overview
+
+* Download: TODO [releases/tag/release-78-1](https://github.com/unicode-org/icu/releases/tag/release-78-1)
+* TODO [Maven: com.ibm.icu / icu4j / version 78.1](https://mvnrepository.com/artifact/com.ibm.icu/icu4j/78.1)
+
+ICU 78 updates to
+[Unicode 17](https://www.unicode.org/versions/Unicode17.0.0/)
+([blog](https://blog.unicode.org/2025/09/unicode-170-release-announcement.html)),
+including new characters and scripts, emoji, collation & IDNA changes, and corresponding APIs and implementations.
+
+It also updates to
+[CLDR 47](https://cldr.unicode.org/downloads/cldr-47)
+([alpha blog](https://blog.unicode.org/2025/09/unicode-cldr-48-alpha-available-for.html))
+locale data with new locales, and various additions and corrections.
+
+TODO: High-level summary
+
+ICU 78 and CLDR 48 are major releases, including a new version of Unicode and major locale data improvements.
+
+TODO status update:
+The Java technology preview implementation of the CLDR MessageFormat 2.0 specification has been updated to incorporate the
+[CLDR 46.1 spec](https://www.unicode.org/reports/tr35/tr35-74/tr35-messageFormat.html)
+plus most but not all of the CLDR 47 changes.
+
+TODO status update:
+The C++ technology preview implementation of MessageFormat 2.0 is not yet quite up to date with CLDR 46.1.
+
+For more details, including migration issues, see below.
+
+Please use the [icu-support mailing list](https://icu.unicode.org/contacts) and/or [find/submit error reports](https://icu.unicode.org/bugs).
+
+### Attention: Future Changes
+
+TODO status update:
+Beginning with ICU 78 (2025-oct):
+
+1. We want to move the minimum required Java version from Java 8 to Java 11.
```

Comment:
This is probably too late to ask for feedback.
So maybe we should sound more decisive 
"We will move to minimum..."

If we are still open to reconsider then add something like "if you have a very convincing argument against this, say it now, or forever hold your peace" :-)

But reverting would be a pain.
We would have to fix the few places where we already used post-11 Java. 
And we might have major problem generating the javadoc and releasing. Because we migrated the taglets and doclets to 11 APIs.

So we would have to build the code with 8 and generate javadoc with 11. Which would be a pain, fighting maven

Reply:
See the TODO